### PR TITLE
Adding Imei Number to Android

### PR DESCRIPTION
**Pull request by vitorhorta**
I am adding the new parameter "Imei Number" to Android platform. It will return an empty string for iOS.

Also, it requires including on AndroidManifest:
uses-permission android:name="android.permission.READ_PHONE_STATE"

I hope it helps!
Thanks for the package!

### CODE REVIEW

**Comment by rebeccahughes** [comment]
Thanks :) I'll test it to see if it will throw an exception if you don't have the read phone state permission, if it's caught then it can be included in the project and just warn the user in the console if they don't have the permission.

**Comment by vitorhorta** [comment]
Did you find any issue with read phone state permission?

**Comment by rebeccahughes** [comment]
Sorry I haven't had a chance to test it yet and I don't have my test device with me

**Comment by athmangude** [comment]
Please merge this branch. Just tested it and it saved the day for me

**Comment by thg303** [comment]
why this is not merged????

**Comment by GantMan** [comment]
not sure why this got closed.  If someone wants to re-submit I will test.  Be sure to mention me by name in the PR.

**Comment by saikiran2907** [comment]
How should I call the parameter to get the imei number

**Comment by mikehardy** [comment]
I think you should ask yourself: can I get the IMEI number? 
The simplest search in the issues list: https://github.com/react-native-device-info/react-native-device-info/issues?q=imei

Says no: https://github.com/react-native-device-info/react-native-device-info/issues/1172#issuecomment-791944303

TL;DR: IMEI access is forbidden as it is a persistent hardware identifier and those are no longer exposed.
